src/elements/TabbedContainer/TabbedContainer.less

```diff
@@ -0,0 +1,54 @@
+.tabs-wrapper {
+	padding: 1em;
+
+	ul.tabs-nav {
```

Comment:
This probably shouldn't be qualified by the `ul` tag, especially since the `List` component is what defines the tag to use. Plain classes are probably ok